src/win/fs.c

```diff
@@ -148,6 +148,72 @@ void uv_fs_init(void) {
 }
 
 
+static int32_t fs__decode_wtf8_char(const char** input) {
+  uint32_t code_point;
+  uint8_t b, b2, b3, b4;
```

Comment:
style nit:
```suggestion
  uint8_t b;
  uint8_t b2;
  uint8_t b3;
  uint8_t b4;
```

(I know the windows parts of the code aren't quite as consistent about this, but it is the general style principle usually observed elsewhere)